translations/monero-core_cat.ts

```diff
@@ -814,32 +814,32 @@
     <message>
         <location filename="../pages/Settings.qml" line="120"/>
         <source>Show seed &amp; keys</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```

Comment:
Please double check before committing, from this point of the text, there are a lot of strings that will result empty, since you removed `type="unfinished"` but didn't provide any translation. Leave `type="unfinished"` if you haven't a translation for a string.

Reply:
Ok, gonna doublecheck everything, plus i did a mistake on the rebase, gonna doublecheck everything and update in a while

Thanks